src/components/crossDevice/CrossDeviceSubmit/index.js

```diff
@@ -26,27 +26,27 @@ class CrossDeviceSubmit extends Component {
     return (
       <div>
         <div className={theme.step}>
-          <h1 className={`${theme.title} ${style.title} ${theme.center}`}>
+          <h1 className={theme.title}>
             Great, that’s everything we need
           </h1>
           <p className={`${theme.center} ${style.submessage}`}>We’re now ready to verify your identity</p>
 
           <ul className={style.uploadList}>
             <li>
               <span className={`${theme.icon} ${style.icon}`}/>
-              <span className={style.listItem}>{documentCopy}</span>
+              {documentCopy}
```

Comment:
not sure if this is a better approach. it has less encapsulation, the style is now being applied to a parent node, rather than a leaf like before.